packages/lib/gpc/test/gpcValibot.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { expect } from "chai";
+import "mocha";
+import { ClosedInterval, GPCProofEntryBoundsCheckConfig } from "../src";
+import {
+  gpcProofEntryBoundsCheckConfigFromJSON,
+  gpcProofEntryBoundsCheckConfigToJSON
+} from "../src/gpcValibot";
+import * as ValibotClosedInterval from "../src/valibot/closedInterval";
+
+// TODO(artwyman): Note the two different forms of import above and calling
+// below.  Is the ValibotClosedInterval form cleaner?
```

Comment:
I think so. Besides the readability benefits, this approach makes the code compartmentalisation clearer, thus making it less likely to miss something when modifying the types/schemas.

Reply:
That was definitely my intent.  Having now done more coding in this form to add additional types, I find it pretty simple to do right.  There's a lot of copy/pasting I don't love, but the type checker is mostly able to tell me if I've done something wrong.  The biggest annoyance I have is that the unusual style of naming the import means that VSCode can't auto-insert missing imports for me.  I think I can live with that since I'm intentionally not exposing any of the Valibot details to any devs outside of the package itself.

Reply:
It strikes me that I could probably make a single module which reexports all of the Valibot* modules by the preferred names in order to make my IDE happier.  I'll experiment with that.